tests/_util/test_functions.py

```diff
@@ -67,3 +69,25 @@ def test_validate_type_set_type():
 def test_object_to_link_by_uid_missing_uids():
     assert {'foo': 'bar'} == object_to_link_by_uid({'foo': 'bar'})
 
+
+def test_rewrite_s3_links_locally():
```

Comment:
Could you add a docstring explaining what this is for? Otherwise it will look bizarre and arbitrary.

Reply:
I documented this in the `functions.py` source code (and updated docs to be more verbose